scripts/run_tests

```diff
@@ -4,17 +4,20 @@ readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
 # shellcheck source=scripts/common
 source "$SCRIPTS_DIR/common"
 
-# For each Rust workspace, first run tests, then run doc tests, then run clippy (turning warnings
-# into errors).
+# For each Rust workspace, first build Wasm modules where appropriate, then run tests, then
+# run doc tests, then run clippy (turning warnings into errors).
 #
 # See:
 # - https://doc.rust-lang.org/cargo/commands/cargo-test.html
 # - https://github.com/rust-lang/rust-clippy.
 
+export RUST_BACKTRACE=1
+
 cargo test --all-targets --manifest-path=./sdk/rust/Cargo.toml
 cargo test --doc --manifest-path=./sdk/rust/Cargo.toml
 cargo clippy --all-targets --manifest-path=./sdk/rust/Cargo.toml -- --deny=warnings
 
+cargo build --target wasm32-unknown-unknown --manifest-path=./examples/Cargo.toml
```

Comment:
This is going to be a pain &ndash; IIUC a clean checkout will always fail `cargo test` for the examples. Not sure what we can do about it though, other than increasing the priority of #541.

```
% cargo test --all-targets --manifest-path=./examples/abitest/tests/Cargo.toml --lib
    Finished dev [unoptimized + debuginfo] target(s) in 5.44s
     Running examples/target/debug/deps/abitest_tests-c1c79200af28eda5

running 1 test
test tests::test_abi ... FAILED

failures:

---- tests::test_abi stdout ----
thread 'tests::test_abi' panicked at 'Module missing: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/libcore/result.rs:1051:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.


failures:
    tests::test_abi
```

Reply:
I'm afraid you're correct, this is something that needs to be resolved sooner rather than later.